### ISSUE TITLE
[BUG] List - remove focus state on list-item on click (web)

### ISSUE THREAD

**Issue by hafstad** [kirbydesign/designsystem#493]
**Describe the bug**
List-item have focused state, on click

**To Reproduce**
1. Go to List examples
2. Click on a list-item

**Kirby version**
 - v0.0.248

**Expected behavior**
List-item **should NOT** have focused state, on click

**Screenshots**
![image](https://user-images.githubusercontent.com/5496706/62619781-e2162800-b917-11e9-8fec-0220ca029d16.png)

**Comment by bommariusser** [comment]
@jb-roa & @PetarRing maybe you have a solution for this fix that you want to fix?

**Comment by bommariusser** [comment]
https://medium.com/@wilkowskidom/how-i-learned-to-stop-worrying-and-love-the-outline-7a35b3b49e7 @hafstad @jakobe @jonasjensen-brolstark maybe something we could be inspired from? 

TLDR: How to keep outline on tab and not on click

**Comment by jakobe** [comment]
Fixed
